src/p4est_io.h

```diff
@@ -113,10 +113,10 @@ p4est_t            *p4est_inflate (sc_MPI_Comm mpicomm,
  * The file header is written to the file as string without null-termination
  * (called string*) and is therefore readable in a text editor.
  *
- * File Header (64 bytes):
```

Comment:
We need to take notes that the documentation depends on bare numbers and change that later.